src/main/java/nl/tudelft/scrumbledore/keybinding/Keybinding.java

```diff
@@ -0,0 +1,95 @@
+package nl.tudelft.scrumbledore.keybinding;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javafx.scene.input.KeyCode;
+import nl.tudelft.scrumbledore.Constants;
+import nl.tudelft.scrumbledore.level.PlayerAction;
+
+/**
+ * Represents a Keybinding in the form of a key value pair.
+ * 
+ * @author Jeroen Meijer
+ */
+public class Keybinding {
+  private Map<KeyCode, PlayerAction> keyMap;
+  private Map<PlayerAction, KeyCode> actionMap;
```

Comment:
I felt that this was the most comprehensive solution and in terms of efficiency, it does not really matter that much. I could indeed make it with one HashMap but it would be less 'clean' in my opinion.